src/handlers/thirdparty.ts

```diff
@@ -49,6 +50,62 @@ export function handleThirdPartyAdded(event: ThirdPartyAdded): void {
   metric.save()
 }
 
+
+export function handleThirdPartyUpdated(event: ThirdPartyUpdated): void {
+  if (!isURNValid(event.params._thirdPartyId)) {
+    log.error('A third party was added with an invalid URN as an id "{}"', [
+      event.params._thirdPartyId
+    ])
+    return
+  }
+
+  let thirdParty = ThirdParty.load(event.params._thirdPartyId)
+
+  if (!thirdParty) {
+    log.error('Invalid third party "{}"', [
+      event.params._thirdPartyId
+    ])
+    return
+  }
+
+  thirdParty.resolver = event.params._resolver
+  thirdParty.rawMetadata = event.params._metadata
+
+  thirdParty.save()
+
+  let eventManagersAddresses = event.params._managers
+  let eventManagersValues = event.params._managerValues
+  let managers = thirdParty.managers
+
+  for (let i = 0; i < event.params._managers.length; i++) {
+    let manager = eventManagersAddresses.pop()
+    let value = eventManagersValues.pop()
+
+    if (value) {
+      managers.push(manager.toHexString())
+      thirdParty.managers = managers
```

Comment:
What if we assign the managers after the for each finishes? IMO it would be cleaner to have the assignments once.

Reply:
I don't want to trust on assemblyscript to be honest. I feel safe by assigning it all the time I manipulate the array. Is it ok for you to keep it as it is?